Source/Scene/PickDepthFramebuffer.js

```diff
@@ -59,16 +51,10 @@ PickDepthFramebuffer.prototype.update = function (
   var width = viewport.width;
   var height = viewport.height;
 
-  if (
-    !defined(this._framebuffer) ||
-    width !== this._depthStencilTexture.width ||
-    height !== this._depthStencilTexture.height
-  ) {
-    destroyResources(this);
-    createResources(this, context);
-  }
+  destroyResources(this);
+  createResources(this, context);
```

Comment:
This will destroy/create the resources every frame.

`framebuffer.update` ought to be sufficient